src/forms/medication-dispense-review.component.tsx

```diff
@@ -208,6 +208,32 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
     setUserCanModify(session?.user && userHasAccess(PRIVILEGE_CREATE_DISPENSE_MODIFY_DETAILS, session.user));
   }, [session]);
 
+  const initialDispenser = useMemo(() => {
+    return medicationDispense?.performer?.[0]?.actor?.reference
+      ? providers?.find((provider) => provider.uuid === medicationDispense.performer[0].actor.reference.split('/')[1])
+      : {
```

Comment:
Looks ok to me, though shouldn't this entire object be undefined if the session?.currentProvider is undefined?

Reply:
Yes, agree with @mseaton ... this should just be undefined in the (unlikely?) case that the there is no current provider associated with the session.